selfdrive/car/subaru/values.py

```diff
@@ -148,6 +148,21 @@ def init_make(self, CP: car.CarParams):
       [SUBARU_VERSION_REQUEST],
       [SUBARU_VERSION_RESPONSE],
     ),
+
+    # Non-OBD requests
+    Request(
+      [StdQueries.TESTER_PRESENT_REQUEST, SUBARU_VERSION_REQUEST],
+      [StdQueries.TESTER_PRESENT_RESPONSE, SUBARU_VERSION_RESPONSE],
+      bus=0,
+      logging=True
```

Comment:
```suggestion
      logging=True,
```

always good to continue commas